### PR TITLE
Back link context sensitive

### PR DESCRIPTION
**Pull request by BenBJSS**

### CODE REVIEW

File: testcafeRunner.js [hunk 1]
```diff
@@ -9,14 +9,14 @@ app.use('/', routes);
 const server = app.listen('1234');
 
 const browserFromArgs = process.argv.slice(2, 3);
-const browserToRun = browserFromArgs.length > 0 ? browserFromArgs : 'chrome:headless';
+const browserToRun = browserFromArgs.length > 0 ? browserFromArgs : 'chrome';
```
Comment (alibombaye) [inline testcafeRunner.js]: can you put this back please

File: testcafeRunner.js [hunk 2]
```diff
@@ -9,14 +9,14 @@ app.use('/', routes);
...
 
 const testFromArgs = process.argv.slice(3, 4);
 const testsToRun = testFromArgs ? `**/*${testFromArgs}*/ui.test.js` : '**/*ui.test.js';
 
-let concurrency = 4;
-let stopOnFirstFail = true;
-let quarantineMode = true;
+let concurrency = 1;
+let stopOnFirstFail = false;
+let quarantineMode = false;
```
Comment (alibombaye) [inline testcafeRunner.js]: and these 😄

File: testcafeRunner.js (hunk 1 above)
Comment (BenBJSS) [reply testcafeRunner.js]: oooooh, yeah *facepalm*

File: testcafeRunner.js (hunk 2 above)
Comment (BenBJSS) [reply testcafeRunner.js]: done

**Pull request merged**